src/tools/segmentation/RectangleScissorsTool.test.js

```diff
@@ -0,0 +1,32 @@
+import RectangleScissorsTool from './RectangleScissorsTool';
```

Comment:
With all of these mocks, I'm not sure we learn much/anything?

Reply:
Yeah, once I refactored the scissors so they don't break api, I removed nearly all the tests that were no longer (maybe were never) irrelevant. We need proper tests for these at some point.